apps/ewallet/lib/ewallet/formatters/amount_formatter.ex

```diff
@@ -3,19 +3,10 @@ defmodule EWallet.AmountFormatter do
   A string formatter for amounts
   """
   def format(amount, subunit_to_unit) do
-    subunit_to_unit
-    |> to_decimals()
-    |> float_to_binary(amount / subunit_to_unit)
-    |> String.replace_trailing(".0", "")
-  end
-
-  defp float_to_binary(decimals, value) do
-    :erlang.float_to_binary(value, [:compact, {:decimals, decimals}])
-  end
+    Decimal.set_context(%Decimal.Context{Decimal.get_context() | precision: 38})
```

Comment:
`Decimal` is available because it's a dependency of `ecto` and `postgrex`.

Now that we're using it directly, can you add `{:decimal, "~> 1.0"}` to `apps/ewallet/mix.exs` so we explicitly require it?

Reply:
Yup, good point.

Reply:
Yeah I was wondering why it was available :)